### ISSUE TITLE
The button tooltip must have the same visuals when button is active and disabled

### ISSUE THREAD

**Issue by joanagrego** [liferay/clay#4992]
When I try to use the TooltipProvider on a disabled button, the tooltip has a different design than when I try to use an active button.

## What are the steps to reproduce?
Use the TooltipProvider on an active button and then on a disabled button. They have different visuals.

## What is the expected result?
Is expected the same tooltip visual both for the active button and for the disabled button.

**Comment by matuzalemsteles** [comment]
I'm removing this issue to be fixed in this release, it's a deeper issue that may require refactoring in our implementation on how we listen for events to add a tooltip, elements with type disabled don't fire events so we'd have to add events on disabled elements.

**Comment by matuzalemsteles** [comment]
I'm investigating this but I see that we have some technical limitations that we can't cover the case when an element has the `disabled=""` property.

Elements with `disabled=""` do not trigger bubbling events and the strategy we used for the `<TooltipProvider />` to be a singleton and be able to monitor any element over some scope that has an attribute with `title` seems to be the only way that we were able to do this.

One way for product teams to bypass this scenario where an element is `disabled` and has a `title`, is to use the CSS class `disabled` and add accessibility property to simulate the disabled state, so we can show the tooltip properly. For example:

```jsx
// instead of
<button className="btn btn-primary" disabled="" title="Open Menu">Click</button>
<Button disabled title="Top">Top</Button>

// use
<button className="btn btn-primary disabled" aria-disabled="true" title="Open Menu">Click</button>
<Button aria-disabled="true" className="disabled" title="Top">Top</Button>
```

In this way, we can support `disabled` elements and are accessible. I'll leave this issue open for others to see, I should also use it to add recommendations on the documentation.
